.github/workflows/continuous-integration.yml

```diff
@@ -73,6 +73,11 @@ jobs:
           ini-values: "zend.assertions=1"
           extensions: "pdo_sqlite"
 
+      - name: Install dependencies
+        run: |
+          composer update
+          composer bin all install
```

Comment:
why installing all ? The psalm bin needs to be installed only for the psalm job.